lib/src/arithmetic/carry_save_mutiplier.dart

```diff
@@ -53,8 +53,6 @@ class CarrySaveMultiplier extends Multiplier {
     clk = addInput('clk', clk);
```

Comment:
why are you overriding `Logic get product` above (can't comment directly on that line here, up on line 33) if it does the same thing as the base `Multiplier`?

Reply:
Cleaned up.